app/helpers/application_helper.rb

```diff
@@ -27,6 +27,10 @@ def markup(text, options={})
     result[:output].to_s.html_safe
   end
 
+  def liquid_text(text)
```

Comment:
is this the best method name for what we are doing here? Does it appropriately describe what this does?

Reply:
Maybe `deliquify_text` would be better?